app/renderer/components/navbar.js

```diff
@@ -22,11 +23,17 @@ module.exports = {
         <nav class="nav-extended">
             <div class="nav-wrapper">
                 <div class="brand-logo main-logo left">
-                <img class="logo-icon" src="resources/logos/gaucho_logo.png"></img>
-                <a>Gaucho</a>
+                    <img class="logo-icon" src="resources/logos/gaucho_logo.png"></img>
+                    <a>Gaucho</a>
+                </div>
+                <div class="tap-target" data-activates="tap-edit">
```

Comment:
The discovery feature could be a separate component, just to keep navbar small and simple